.github/workflows/deploy-admin.yml

```
@@ -30,11 +41,19 @@ jobs:
     - name: 프로젝트 빌드
-      run: ./gradlew clean build -x test
+      run: ./gradlew clean :application-admin:bootJar -x test
+
+    - name: Prepare deployment package
+      run: |
+        rm -rf deploy && mkdir deploy
+        cp application-admin/build/libs/application-admin-*.jar deploy/application-admin.jar
+        cp scripts/start-admin.sh scripts/stop-admin.sh appspec-admin.yml deploy/appspec.yml
+        cd deploy
+        zip -r ../${{ env.PROJECT_NAME }}-${{ env.DIVISION }}-${{ github.sha }}.zip .
     
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)